app/pricing/FormattersComponents.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import * as React from "react";
+
+const SERVER_LOCALE = "en-US";
+
+const IntlNumberFormat = ({
+  options,
+  value,
+}: {
+  options: Intl.NumberFormatOptions;
+  value: number;
+}) => {
+  const optionsRef = React.useRef(options);
+  const [formatter, setFormatter] = React.useState(
+    () => new Intl.NumberFormat(SERVER_LOCALE, optionsRef.current),
+  );
+
+  React.useEffect(() => {
+    setFormatter(() => new Intl.NumberFormat(undefined, optionsRef.current));
```

Comment:
As I said it's inneficient to set the formatter and not the value.

Reply:
You said inefficient. I said it's a NIT (nitpicking). You you double down on it. Well it won't make it move fast